src/plugins/intel_gpu/tests/fusions/fusion_validity_test.cpp

```diff
@@ -0,0 +1,127 @@
+    auto p = GetParam();
+    create_topologies(
+        // Fused eltwise contains format mistmatch between data input of resample and fused eltwise input
+        input_layout("input", get_input_layout(p)),
+        data("eltwise_data", get_mem(get_default_layout(p), -10, 10)),
+        resample("resample_opt", "input", p.out_shape, 1, p.type),
+        eltwise("eltwise", { "eltwise_data", "resample_opt" }, eltwise_mode::sum),
+        reorder("reorder_bfyx", "eltwise", p.output_format, data_types::f32)
+    );
+
+    implementation_desc resample_impl = { p.input_format, "resample_opt" };
+    bo_fused.set_option(build_option::force_implementations({ { "resample_opt", resample_impl } }));
+    implementation_desc ref_resample_impl = { p.input_format, "resample_ref" };
+    bo_not_fused.set_option(build_option::force_implementations({ { "resample_opt", ref_resample_impl } }));
+    implementation_desc ref_eltwise = { p.input_format, "" };
+    bo_not_fused.set_option(build_option::force_implementations({ { "eltwise_data", ref_eltwise } }));
+
+    tolerance = 1e-5f;
+    execute(p);
+}
+
+INSTANTIATE_TEST_SUITE_P(validate_fusings_gpu, resample_eltwise_fusing, ::testing::ValuesIn(std::vector<fusing_test_params>{
+    fusing_test_params{ CASE_RESAMPLE_FP16_1, 3, 4 },
+    fusing_test_params{ CASE_RESAMPLE_FP16_2, 3, 4 },
+    fusing_test_params{ CASE_RESAMPLE_FP32_1, 3, 4 },
+    fusing_test_params{ CASE_RESAMPLE_FP32_2, 3, 4 }
+}));
+
+
+class resample_multiple_fusing : public PrimitiveFusingTest {};
+TEST_P(resample_multiple_fusing, basic) {
+    auto p = GetParam();
+    create_topologies(
+        // Multiple fused prims which contains format mismatch
+        input_layout("input", get_input_layout(p)),
+        data("scale_data", get_mem(get_per_channel_layout(p), -10, 10)),
+        data("eltwise_data", get_mem(get_default_layout(p), -10, 10)),
+        resample("resample_prim", "input", p.out_shape, p.in_shape.feature[0], p.type),
+        scale("scale", "resample_prim", "scale_data"),
+        activation("activation", "scale", activation_func::abs),
+        eltwise("eltwise", { "activation", "eltwise_data" }, eltwise_mode::sum),
+        reorder("reorder_bfyx", "eltwise", p.default_format, data_types::f32)
+    );
+
+    implementation_desc resample_impl = { p.input_format, "resample_opt" };
+    bo_fused.set_option(build_option::force_implementations({ { "resample_prim", resample_impl } }));
+    implementation_desc ref_resample_impl = { p.input_format, "resample_ref" };
+    bo_not_fused.set_option(build_option::force_implementations({ { "resample_prim", ref_resample_impl } }));
+    implementation_desc ref_eltwise = { p.input_format, "" };
+    bo_not_fused.set_option(build_option::force_implementations({ { "eltwise_data", ref_eltwise } }));
+
+    tolerance = 1e-5f;
+    execute(p);
+}
+
+INSTANTIATE_TEST_SUITE_P(validate_fusings_gpu, resample_multiple_fusing, ::testing::ValuesIn(std::vector<fusing_test_params>{
```

Comment:
`validate_fusings_gpu` and `resample_multiple_fusing` sound very general. I think names can be more specific.

Reply:
Applied